src/components/SignUpForm/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./styles.scss";
+
+export default function Form() {
+  return (
+    <div className="signUp-form">
+      Sign up
+      <Link to="/registration-success" className="btn btn-submit">
+        Submit
+      </Link>
+    </div>
```

Comment:
```suggestion
    <div className="sign-up-form">
      Sign up
      <Link to="/registration-success" className="sign-up-submit-btn">
        Submit
      </Link>
    </div>
```
Since there could be many submit buttons, naming `btn-submit` would further make some confusion.